versioning/scripts/dockerfiles/main.go

```diff
@@ -250,6 +255,9 @@ func main() {
 			failureCount += verifyDockerfiles(version, filepath.Join(*defaultTemplateDirPtr, version.TemplateSubDir), *tmpl)
 			failureCount += verifyCopiedFiles(version, filepath.Join(*defaultTemplateDirPtr, version.TemplateSubDir))
 		} else {
+			if *addDir {
```

Comment:
Comparing on the comments from `copyFiles` func, I think we should move this to the `writeDockerfile` to be consistent on what the `copyFiles` function does.